### ISSUE TITLE
Values in the `prestige_xp` lists

### ISSUE THREAD

**Issue by johndoknjas** [brooke-gill/pit#32]
Hi, I noticed that for the `prestige_xp` lists, the values of each element may be slightly too high (not entirely sure though):

       `xp_map = [15, 30, 50, 75, 125, 300, 600, 800, 900, 1000, 1200, 1500, 0];
        prestiges = [100, 110, 120, 130, 140, 150, 175, 200, 250, 300, 400, 500, 600, 700, 800, 900, 1000, 1200, 1400, 1600, 1800, 2000, 2400, 2800, 3200, 3600, 4000, 4500, 5000, 7500, 10000, 10100, 10100, 10100, 10100, 10100, 20000, 30000, 40000, 50000, 75000, 100000, 125000, 150000, 175000, 200000, 300000, 500000, 1000000, 5000000, 10000000,];
        prestige_xp = [65950, 138510, 217680, 303430, 395760, 494700, 610140, 742040, 906930, 1104780, 1368580, 1698330, 2094030, 2555680, 3083280, 3676830, 4336330, 5127730, 6051030, 7106230, 8293330, 9612330, 11195130, 13041730, 15152130, 17526330, 20164330, 23132080, 26429580, 31375830, 37970830, 44631780, 51292730, 57953680, 64614630, 71275580, 84465580, 104250580, 130630580, 163605580, 213068080, 279018080, 361455580, 460380580,575793080, 707693080, 905543080, 1235293080, 1894793080, 5192293080,11787293080,];`

E.g., for the first element of `prestige_xp`, 65950 is what you get by multiplying each value in `xp_map` by 10 and then summing the products. However, since players start at level 1, this sum is too great (by counting 15xp for level 0 --> 1). So instead the first element of `prestige_xp` should be 65935, which is obtained by 9(15)+10(30)+10(50)+...+10(1500).

I may be missing something though, since I noticed in `oldtracker.html` that 65935 used to be the first element's value, but it was commented out and replaced with 65950.

**Comment by brooke-gill** [comment]
Hi John,
The `prestige_xp` variable calculates the amount of XP a player has by taking the amount of XP required to reach level 120, and then subtracting XP repeatedly (counts downward) until the XP is less than the amount the player has. If you're curious, you can see this on line 255 of `tracker.html`:

```js
x_120level = prestige_xp[x_prestige];

while (x_120level > experience) {
    x_level = x_level - 1;
    x_120level = x_120level - Math.ceil((xp_map[Math.floor(x_level / 10)] * prestiges[x_prestige]) / 100);
}
```

The reason that this is done is to have an accurate way to track XP. The leading theory to why many XP-to-level conversions are slightly wrong is that Pit gives players a differing quantity of bonus XP when they prestige (in an addition to the level-up from 0 → 1 each prestige). While the difference is virtually unnoticable, the error that occurs with trackers that count up instead of down can be seen by checking certain players, such as `trogador9555`, a player with the current level `[I-73]`. An XP tracker that counts using the traditional method, such as [Pit Panda](https://pitpanda.rocks/players/trogador9555), will display the incorrect level of `[I-74]` because it doesn't take into account this bonus XP.

TL;DR: You can do it the way `oldtracker.html` did – but some edge cases will be off by a level.

**Comment by johndoknjas** [comment]
Ah I see, thanks for the info. So when a player prestiges, hypixel gives them an amount of bonus xp that automatically takes them from level 0 to 1?
